mappings/net/minecraft/block/Block.mapping

```diff
@@ -207,7 +207,7 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 0 stateId
 	METHOD method_9533 shouldDropItemsOnExplosion (Lnet/minecraft/class_1927;)Z
 		ARG 1 explosion
-	METHOD method_9534 activate (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;Lnet/minecraft/class_3965;)Z
+	METHOD method_9534 use (Lnet/minecraft/class_2680;Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;Lnet/minecraft/class_3965;)Z
```

Comment:
Whats the reasoning behind this? Its not wrong, but activate makes a bit more sense as use is a bit more of a generic term imo?

Reply:
it is when player press use key on the block

Reply:
`use` is more consistent with the keybinding and the item methods. You're also not "activating" a chest, you're *using* it. Activating to me sounds more like activating a piston or a dispenser.